bin/gen.py

```diff
@@ -45,16 +45,17 @@ def warning(self, lno: int, err_msg: str) -> None:
 def main(f: TextIO, reporter: Reporter, by_epd: Dict[str, List[str]], shortest_by_name: Dict[str, int]) -> None:
     prev_eco = ""
     prev_name = ""
-
+   
+    expected_header = ["eco", "name", "pgn"]
     for lno, line in enumerate(f, 1):
-        cols = line.rstrip("\n").split("\t")
+        cols = line.rstrip("\n").split("\t")[:3]
 
-        if len(cols) != 3:
-            reporter.error(lno, f"expected 3 columns, got {len(cols)}")
+        if len(cols) < 3:
+            reporter.error(lno, f"expected at least 3 columns, got {len(cols)}")
```

Comment:
What's the idea here, i.e., why allow additional colummns?

Reply:
I believe this is for b when I collapse all generated into all.tsv

Reply:
Sorry, I don't understand. Can you please explain a bit more?

Reply:
Take a peek at the Makefile. I'm now building an all.tsv that combines the others. Though now I'm wondering if this particular change is for code that didn't make it in....

Reply:
Are there any changes I should make?